tox.ini

```diff
@@ -1,37 +1,78 @@
-# Copyright 2021 Canonical Ltd.
+# Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
 
 [flake8]
 max-line-length = 100
-ignore = W503
-copyright-check = True
-copyright-author = Canonical Ltd.
-select = E,F,W,C
 
 [tox]
 skipsdist = True
+skip_missing_interpreters = True
+envlist = fmt, lint, unit, integration
+
+[vars]
+all_path = {[vars]src_path} {[vars]tst_path}
+src_path = {toxinidir}/src/
+tst_path = {toxinidir}/tests/
 
 [testenv]
-passenv = HOME
+passenv =
+    PYTHONPATH
+    CHARM_BUILD_DIR
+    MODEL_SETTINGS
+    KUBECONFIG
 setenv =
-    PYTHONPATH={toxinidir}/src:{toxinidir}/lib
+    PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}
+    PYTHONBREAKPOINT=ipdb.set_trace
+    PY_COLORS=1
+
+[testenv:update-requirements]
```

Comment:
If we keep the env (see note below), then lets either rename this env or rename the requirements to `requirements-update-requirements.*` so it is `requirements-ENVNAME.*`

Reply:
I used the option above